File: Genral_All_Controls/StandaloneExpressionEditorForm/StandaloneExpressionEditorFormCS/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using Telerik.WinControls.UI;

namespace StandaloneRadExpressionEdtiorForm
{
    public partial class MainForm : Form
    {
        public MainForm()
        {
            InitializeComponent();
            this.radButton1.Click += radButton1_Click;
        }

        void radButton1_Click(object sender, EventArgs e)
        {
            StandaloneExpressionEditorForm form = new StandaloneExpressionEditorForm();
            if (form.ShowDialog(this.radTextBox1.Text) == System.Windows.Forms.DialogResult.OK)
            {
                this.radTextBox2.Text = form.ReturnValue;
            }
        }
    }
}

File: Genral_All_Controls/StandaloneExpressionEditorForm/StandaloneExpressionEditorFormVB/MainForm.vb
Imports Telerik.WinControls.UI

Partial Public Class MainForm

    Public Sub New()
        InitializeComponent()
        AddHandler Me.RadButton1.Click, AddressOf radButton1_Click

    End Sub

    Private Sub radButton1_Click(sender As Object, e As EventArgs)
        Dim form As New StandaloneExpressionEditorForm()
        If form.ShowDialog(Me.RadTextBox1.Text) = System.Windows.Forms.DialogResult.OK Then
            Me.RadTextBox2.Text = form.ReturnValue
        End If
    End Sub
End Class
